stairway/src/main/java/bio/terra/stairway/FlightFilter.java

```diff
@@ -375,6 +412,44 @@ public static FlightFilterPredicate makePredicateSubmitTime(
       return makePredicateFlight("submit_time", op, timestamp, Datatype.TIMESTAMP);
     }
 
+    @Override
+    public List<Value> getValues() throws JsonProcessingException {
+      return List.of(switch (type) {
+        case INPUT -> getInputValues();
+        case FLIGHT -> getFlightValues();
+      });
+    }
+
+    private Value getInputValues() throws JsonProcessingException {
+      return switch (datatype) {
+      case LIST -> {
+        List<String> values = new ArrayList<>();
+        for (Object obj : ((List<?>) value)) {
+          // Need to double encode in order for strings to match
+          values.add(
+                  StairwayMapper.getObjectMapper()
+                          .writeValueAsString(StairwayMapper.getObjectMapper().writeValueAsString(obj)));
+        }
+        yield new Value(this, "[" + String.join(",", values) + "]");
+      }
+      case NULL -> new Value();
+      default -> new Value(this, StairwayMapper.getObjectMapper().writeValueAsString(value));
+      };
+    }
+
+    /**
+     * Get the parameter value for substitution into the prepared statement.
+     */
+    private Value getFlightValues() throws JsonProcessingException {
+      return switch (datatype) {
+        case STRING -> new Value(this, (String) value);
+        case LIST -> new Value(this, pgJsonMapper.writeValueAsString(value));
```

Comment:
Any reason not to use `StairwayMapper.getObjectMapper()` here?

Reply:
I don't know, my goal was only to refactor so in this case I preserved what the original code was doing. Maybe @ddietterich could say for sure?

Reply:
I suppose we should just leave that for now and we can come back to fix if needed